jws/jws.go

```diff
@@ -29,15 +29,14 @@ type Header struct {
 	// Key ID Header Parameter https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.4
 	KID string `json:"kid,omitempty"`
 	// Type Header Parameter https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.9
-	Type string `json:"typ,omitempty"`
+	TYP string `json:"typ,omitempty"`
 }
 
 type JWSPayload any
 
 // Base64UrlEncode returns the base64url encoded header.
 func (j Header) Base64UrlEncode() string {
-	jsonHeader := map[string]string{"alg": j.ALG, "kid": j.KID}
-	bytes, _ := json.Marshal(jsonHeader)
+	bytes, _ := json.Marshal(j)
```

Comment:
would be a good idea to deal with that _ (error) at some point.

Reply:
Fixed here https://github.com/TBD54566975/web5-go/pull/42

Reply:
fixed